cleanlab/rank.py

```diff
@@ -26,12 +26,8 @@
 Note: multi-label classification is not supported by most methods in this module,
```

Comment:
Can we delete all the lines in this file that say:

`**Caution**: `pred_probs` from your model must be out-of-sample!`

Even these sections in filter.py can be toned down a little

Reply:
I've done this for you so I can get this merged now